database/auth.go

```diff
@@ -52,6 +53,41 @@ func (db *Db) saveAccounts(paramsNumber int, accounts []types.Account) error {
 	return nil
 }
 
+func (db *Db) SaveVestingAccounts(paramsNumber int, vestingAccount []types.VestingAccount) error {
+	if len(vestingAccount) == 0 {
+		return nil
+	}
+	stmt := `INSERT INTO vesting_account (address, original_vesting, end_time, start_time, vesting_periods) VALUES `
+	var params []interface{}
+
+	for i, account := range vestingAccount {
+		ai := i * paramsNumber
+		stmt += fmt.Sprintf("($%d,$%d,$%d,$%d,$%d),", ai+1, ai+2, ai+3, ai+4, ai+5)
+
+		originalVestingBz, err := json.Marshal(account.OriginalVesting)
+		if err != nil {
+			return err
+		}
+
+		VestingPeriodsBz, err := json.Marshal(account.VestingPeriods)
+		if err != nil {
+			return err
+		}
+
+		params = append(params, account.Address, string(originalVestingBz), account.EndTime, account.StartTime, string(VestingPeriodsBz))
+	}
+
+	stmt = stmt[:len(stmt)-1]
+	stmt += `ON CONFLICT (address) DO NOTHING`
+
+	_, err := db.Sql.Exec(stmt, params...)
```

Comment:
I tested it on my local and i get an error:
```12:27PM ERR error while handling genesis err="error while storing genesis vesting accounts: pq: date/time field value out of range: \"1756649664\"" module=auth```

Reply:
oh yes I forgot to change the schema, it should be saved at TEXT. just pushed the fix now.